src/pages/hegel/guides/science-of-logic-introduction.mdx

```diff
@@ -0,0 +1,184 @@
+
+The cause for this error, Hegel diagnoses, lies in the fact that the determinations or categories of the understanding were never given an independent examination and immanent deduction but were simply taken for granted.  
+
+#### On the Absurdity of Knowledge Based on Appearances (Niklas)
+
+Further observations may be drawn regarding the absurdity of truth vis-à-vis appearances (what we can call the double-standard of knowledge) in inspecting its implicit elements. When it is posited that knowledge is split by way of subject matters, and that of certain subject matters one can know nothing about (such as things-in-themselves) but others one can have knowledge (such as what appears in experience), knowledge remains the common element in both. The incoherence may be traced to an unstated third element that orchestrates the mediation between knowledge based on appearances and knowledge as such. This third element can be either one of the two extremes (knowledge based on appearances or knowledge as such) or something else entirely. It seems that, according to Hegel, if transcendental idealism is taken at face value, one subscribes to the view that knowledge based in appearances grounds both itself and knowledge as such, yet Hegel's criticism is that this precisely leads to absurdity which subsequently must mean that Kant merely takes for granted either that knowledge as such grounds both itself and knowledge based in appearances or that there is another third, yet unknown, common denominator. The latter seems to align well with Kant's own thinking in the _First Critique_:
+
+> All that seems necessary for an introduction or a preliminary is that there are two stems of human cognition, which may perhaps arise from a common but to us unknown root, namely sensibility and understanding, through the first of which objects are given to us, but through the second of which they are thought (Kant, 1998, 135/A15).
+
+In fairness to Kant, however, more emphasis is placed on pure concepts as the argument of transcendental idealism develops. Kant, in contrast to Hume, places pure categories or pure concepts in the purview of reason such that these cannot have their validity or truth established by sensuous reality, but that, rather, they serve to establish the validity and truth of sensuous reality. There are also a number of other key concepts&mdash;called _concepts of reason_&mdash;that never directly interact with sensuous reality, such as ethical principles or the concept of world (see Kant, 1998, 394/A311). Indeed, concepts of reason are in this way genuinely unconditional. With these things in mind, the thrust of Hegel's criticism is somewhat blunted, yet his real target is the fact that transcendental idealism has invalid or weak means to establish the truth of the very tools of cognition it purports to use to establish truth _tout court_. 
```

Comment:
"With these things in mind, the thrust of Hegel's criticism is somewhat blunted, yet his real target is the fact that transcendental idealism has invalid or weak means to establish the truth of the very tools of cognition it purports to use to establish truth _tout court_." It would be good to flesh out why "Hegel's criticism is...blunted" but the damage done to his "real target" still hits. Unpacking this a bit would make for easier reading.

Reply:
Ok! Added!